### PR TITLE
re-enable the build defaulter plugin by default

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
[testextended][extended:core(builds)]

**Comment by bparees** [comment]
@deads2k ptal

@oatmealraisin @jupierce @php-coder @PI-Victor fyi.

**Comment by bparees** [comment]
[test]

File: pkg/cmd/server/start/admission.go [hunk 1]
```diff
@@ -74,7 +74,9 @@ var (
 		"PodNodeConstraints",
 		overrideapi.PluginName,
 		imagepolicy.PluginName,
-		"BuildDefaults",
+		// TODO: remove the log setting logic from the build defaulter and make this
+		// default off again.
+		//"BuildDefaults",
```
Comment (deads2k) [inline pkg/cmd/server/start/admission.go]: Move this up to the `defaultOnPlugins` stanza.  This should have failed unit tests.  Did it pass?

Comment (bparees) [inline pkg/cmd/server/start/admission.go]: i don't test locally :)  was going to let the bot tell me.

Comment (bparees) [inline pkg/cmd/server/start/admission.go]: fixed.

File: pkg/cmd/server/start/admission.go [hunk 2]
```diff
@@ -74,7 +78,9 @@ var (
 		"PodNodeConstraints",
 		overrideapi.PluginName,
 		imagepolicy.PluginName,
-		"BuildDefaults",
+		// TODO: remove the log setting logic from the build defaulter and make this
```
Comment (deads2k) [inline pkg/cmd/server/start/admission.go]: this section is now superfluous.

File: pkg/cmd/server/start/admission.go [hunk 3]
```diff
@@ -50,6 +50,10 @@ var (
 		"OriginNamespaceLifecycle",
 		"openshift.io/JenkinsBootstrapper",
 		"BuildByStrategy",
+		// TODO: remove the log setting logic from the build defaulter and make this
+		// default off again.
+		//"BuildDefaults",
```
Comment (deads2k) [inline pkg/cmd/server/start/admission.go]: cruft

**Comment by deads2k** [comment]
nits, lgtm.

File: pkg/cmd/server/start/admission.go [hunk 4]
```diff
@@ -50,6 +50,9 @@ var (
 		"OriginNamespaceLifecycle",
 		"openshift.io/JenkinsBootstrapper",
 		"BuildByStrategy",
+		// TODO: remove the log setting logic from the build defaulter and make this
+		// default off again.
+		"BuildDefaults",
```
Comment (bparees) [inline pkg/cmd/server/start/admission.go]: @deads2k better?

Comment (deads2k) [inline pkg/cmd/server/start/admission.go]: > @deads2k better?

a thing of beauty :)

**Comment by bparees** [comment]
[merge]

**Pull request merged**